docs/section19/19-9.md

```diff
@@ -0,0 +1,34 @@
+# Section 19 - Rule 19-9     
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-9     
+**Rule Description:** Air economizers shall not be included in baseline HVAC Systems 1, 2, 9, and 10.    
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.6      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline HVAC system types 1, 2, 9, and 10.
+
+**Function Calls:**  
+1. get_baseline_system_types  
+
+## Rule Logic:   
+**Applicability Check 1**   
+- Create ditionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMR)`  
+- Check if B-RMR is modeled with at least one air-side system that is Type-1,1a,1b,1c,2,9,9b,10, if yes then carry on, if no then NOT_APPLICABLE for the project: `for sys_type in ["SYS-1", "SYS-1a", "SYS-1b", "SYS-1c", "SYS-2", "SYS-9", "SYS-9b", "SYS-10"]):`
+    - Check if values are not none for system type key in the dictionary, if it does not equal false then the values are not none and there are applicable system types in the project: `if bool(baseline_system_types_dict.get(sys_type)) != False: is_there_applicable_sys_type = TRUE`
+- Check if is_there_applicable_sys_type == True which means there are applicable system types in the project, if yes then carry on, if no then NOT_APPLICABLE for the project: `if  is_there_applicable_sys_type == True:`  
+    - For each hvac system in the B_RMR: `for hvac in B_RMR...HeatingVentilatingAirConditioningSystem:` 
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `If any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-1", "SYS-1a", "SYS-1b", "SYS-1c", "SYS-2", "SYS-9", "SYS-9b", "SYS-10"]): `                    
+            - **Rule Assertion:** 
+            - Case 1: If there is NOT an AirEconomizer object associated with the fan (equals Null) or if the aireconomizer type is fixed fraction then pass: `if hvac.fan_system.AirEconomizer == Null OR hvac.fan_system.AirEconomizer.type == "FIXED_FRACTION": outcome = "PASS"`  
+            - Case 2: Else, fail: `Else: outcome = "Fail"`  
```

Comment:
Do we want to indicate that the rule passes in all cases if "allow more stringent" flag is on? This also applies to the 19-8 (DCV).

Reply:
Decided against allowing projects to pass when an economizer is modeled when not required even when AHJ_RA_compare == True.